cmd/daemon/kubernetes/kubernetes.go

```diff
@@ -34,10 +34,11 @@ func NewClient(kubeConfigPath string, moduloCrashReportNotif float64, e Exporter
 }
 
 func isCorrectlyAnnotated(annotations map[string]string) bool {
-	if !(annotations["lunarway.com/controlled-by-release-manager"] == "true") &&
-		annotations["lunarway.com/artifact-id"] == "" &&
-		annotations["lunarway.com/author"] == "" {
-		return false
+
```

Comment:
Remove new line?

Reply:
Also applicable for kubernetes_test.go

Reply:
I usually add a newline before EOF because it plays well with git diffs. my go-land go formatter doesn't complain about them. is this not "go idiomatic"?

Reply:
Its not the EOF newmine but the function opening I think. :)

Reply:
ahh :-)

Reply:
I remember pointing the function opening. weird :)

Reply:
The love of rebasing during review @jweibel22 ? 🙈